packages/fleather/lib/src/widgets/editable_text_block.dart

```diff
@@ -364,11 +361,15 @@ class _CheckboxPoint extends StatefulWidget {
 class _CheckboxPointState extends State<_CheckboxPoint> {
   @override
   Widget build(BuildContext context) {
-    return Checkbox(
-      value: widget.value,
-      onChanged:
-          widget.enabled ? (value) => widget.onChanged(!widget.value) : null,
-      splashRadius: 0,
+    return Align(
```

Comment:
Indeed, 
getting rid of the splash --> better
leadings are not centered but aligned to end --> better

width or height have no effect though right?

Reply:
It made difference when I was testing it.🤔 Let me check again. @amantoux

Reply:
Using `alignment: AlignmentDirectional.topEnd, padding: const EdgeInsetsDirectional.only(end: 8.0),` in `Container` we get this:
<img width="129" alt="Screenshot 2023-05-11 at 23 51 54" src="https://github.com/fleather-editor/fleather/assets/8766492/3cb5b0ee-30f4-4026-93dc-389f81e82f26">

And using `height: 20, width: 20, alignment: AlignmentDirectional.topStart` in `Container` we get this:
<img width="117" alt="Screenshot 2023-05-11 at 23 54 13" src="https://github.com/fleather-editor/fleather/assets/8766492/2e62f7a5-18f3-4123-a9f0-733efb77598b">

I think second one is better. WDYT? @amantoux

Reply:
Second seems better indeed
Instead empirically forcing the size of the checkbox, I propose to create a custom one so as rely on a position rather than a size @Amir-P